src/mom5/ocean_core/ocean_sbc.F90

```diff
@@ -1102,10 +1113,12 @@ subroutine ocean_sbc_init(Grid, Domain, Time, T_prog, T_diag, &
 #if defined(ACCESS_CM) || defined(ACCESS_OM)
   allocate ( Ocean_sfc%gradient (isc_bnd:iec_bnd,jsc_bnd:jec_bnd,2))
   allocate ( sslope(isc:iec, jsc:jec, 2) )
-  allocate ( aice(isc:iec, jsc:jec) )
+  allocate ( aice(isd:ied, jsd:jed) )
```

Comment:
Just checking you did mean to change this from compute domain

Reply:
Yes. That's how it is declared in WOMBAT. If it is needed at some later date I felt it would be more consistent to carry around a halo version. Might also be needed for the output diagnostic.